### ISSUE TITLE
Don't flag a missing accessibilitySummary as serious error

### ISSUE THREAD

**Issue by mkraetke** [daisy/ace#378]
- ACE 1.2.6
- v16.15.1
- Windows 11 with Cygwin

If the `accessibilitySummary` is missing, ACE reports this as "serious" error. However, the [EPUB Accessibility Techniques 1.1](https://www.w3.org/TR/epub-a11y-tech-11/#example-accessibility-summary-that-complements-the-conformance-statement) clarify:
> EPUB creators should not include an accessibility summary when they have nothing more to add to the conformance claim and other discovery metadata.

The [[EPUB Accessibility 1.1](https://www.w3.org/TR/epub-a11y-11/](https://www.w3.org/TR/epub-a11y-11/#sec-disc-package)) state that the summary should but not must be included:
> EPUB publications SHOULD include the following [[schema-org](https://www.w3.org/TR/epub-a11y-11/#bib-schema-org)] accessibility metadata.

We experienced that the report confuses publishers and some accessibility agencies alike. They think they must have the summary in the EPUB even if EPUB includes all other accessibility metadata. So the summary would just repeat the information of the other metadata, which is redundant. So I'd like suggest to lower the error level from `serious` to `minor`.

**Comment by danielweck** [comment]
Indeed, the specification was recently updated to relax the `accessibilitySummary` rules. Ace must now be updated accordingly.

**Comment by danielweck** [comment]
Note that EPUB Accessibility 1.1 "Conformance and Discoverability Requirements for EPUB publications" is a W3C Candidate Recommendation (latest draft dated `08 September 2022`). In Ace we must be careful not implementing features that are not final, but based on my observation of the working group discussions, I am confident that they have now settled definitely on the `accessibilitySummary` decision. Is that right @avneeshsingh ?

https://www.w3.org/TR/epub-a11y-11/#change-latest

> Substantive changes since Candidate Recommendation
> 12-Aug-2022: Changed accessibilitySummary from a required to a recommended property and updated its definition to focus less on repeating information available in the other metadata.

**Comment by avneeshsingh** [comment]
Yes AccessibilitySummary is a SHOULD now instead of MUST

 

With regards

Avneesh

**Comment by danielweck** [comment]
Thank you Avneesh. The Ace implementation currently tracks the latest CR Candidate Recommendation of version 1.1 of the specification, so let's fix this as soon as possible in the next release.
